### DISCUSSION TITLE
Instagram password

### DISCUSSION THREAD

**Discussion by Badshahboyannu** [akhatkulov/InstaCracker-CLI#16] [General]
Password

**Comment by Vijayking2527** [comment]
Instagram password cracker

**Comment by Vijayking2527** [comment]
Password hack

**Comment by D4mZ8K** [comment]
hey im a new user in github and i was just wondering how do i run instacracker? i am on windows but i have kali linux in a vm and also just a kali linux terminal (installed it from microsoft store). so how do i run it?

**Comment by akhatkulov** [comment]
you need learn about, how to run python code...
